crates/harp/src/size.rs

```diff
@@ -184,14 +184,16 @@ fn obj_size_tree(
                         seen,
                         depth + 1,
                     );
-                    size += obj_size_tree(
-                        unsafe { libr::CAR(cons) },
-                        base_env,
-                        sizeof_node,
-                        sizeof_vector,
-                        seen,
-                        depth + 1,
-                    );
+                    if !is_immediate_binding(cons) {
```

Comment:
I think you can still compute the size by looking at the `TAG` and using `getVar()`.

Reply:
I somehow understood that since the value is inlined, the size is already computed when we add the node size: https://github.com/posit-dev/ark/blob/eb1260c5f8a7c1c87cfd0c738028a362bbf73a64/crates/harp/src/size.rs#L178-L179

Ie, the value is using the same space as the *SEXPREC would use.

Am I missing something?

Reply:
oh no you're totally right! Can you please add a comment?